README.md

```diff
@@ -4,4 +4,6 @@ This plugin wraps [ArcWelderLib](https://github.com/FormerLurker/ArcWelderLib) b
 
 Inspired by on a postprocessing script by [yysh12](https://github.com/yysh12)
 
-For further informatio about ArcWelder antistutter, see https://plugins.octoprint.org/plugins/arc_welder
\ No newline at end of file
+For further informations about ArcWelder antistutter, see https://plugins.octoprint.org/plugins/arc_welder
```

Comment:
Thanks for ficing this typo, however `information` has on plural form in english. `For further information about ...`